modules/custom/openy_activity_finder/src/Controller/ActivityFinderController.php

```diff
@@ -94,15 +94,4 @@ public function redirectToRegister(Request $request, $log) {
     return new TrustedRedirectResponse($url, 301);
   }
 
-  /**
```

Comment:
@aaronjchristian this is obviously accidental removal of the code added later. Please pull in the latest changes and do not overwrite them.

Reply:
Possible a bad merge here.